src/screens/Connectors/PaymentProcessor/ConnectPayPalFlow/PayPalFlowUtils.res

```diff
@@ -104,12 +99,10 @@ let generateConnectorPayloadPayPal = (
       ("connector_label", connectorLabel->JSON.Encode.string),
     ]->LogicUtils.getJsonFromArrayOfJson
 
-  generateInitialValuesDict(
-    ~values={initialValues},
-    ~connector,
-    ~bodyType,
-    ~isPayoutFlow=false,
-  )->ignoreFields(connectorId, connectorIgnoredField)
+  generateInitialValuesDict(~values={initialValues}, ~connector, ~bodyType)->ignoreFields(
+    connectorId,
+    connectorIgnoredField,
```

Comment:
here you should pass the prop of connector type, introduce a new prop called connector type in `handleObjectResponse` method, as this is called from `ConnectorHome` and `PayoutProcessorHome`